modules/gce_docker_registry_cache/cloud-config.yml.tpl

```diff
@@ -0,0 +1,20 @@
+#cloud-config
+# vim:filetype=yaml
+
+write_files:
+- content: '${base64encode(docker_config)}'
+  encoding: b64
+  path: /etc/default/docker
+- content: '${base64encode(docker_registry_config)}'
+  encoding: b64
+  path: /etc/docker/registry/config.yml
+- content: '${base64encode(file("${here}/cloud-init.bash"))}'
+  encoding: b64
+  path: /var/lib/cloud/scripts/per-instance/99-travis-cloud-init
+  permissions: '0750'
+- content: '${base64encode(syslog_address)}'
+  encoding: b64
+  path: /var/tmp/travis-run.d/syslog-address
+- content: '${base64encode(file("${here}/daemon-direct-lvm.json"))}'
```

Comment:
Is direct-lvm storage helpful in this case? My understanding was that all blob storage was external.